### PR TITLE
Add new mobile team members

### PR DESCRIPTION
**Pull request by enahum**

### CODE REVIEW

**Review by jasonblais — APPROVED** [review]

File: site/content/contribute/getting-started/core-committers.md [hunk 1]
```diff
@@ -26,7 +26,7 @@ Below is the list of core committers working on Mattermost:
     - Dev areas: Mobile / Desktop and Web apps
 - **<a name="elias.nahum">Elias Nahum</a>**
     - @elias on [community.mattermost.com](https://community.mattermost.com/core/messages/@elias) and [@enahum](https://github.com/enahum) on GitHub
-    - Dev areas: Mobile / Desktop and Web apps
+    - Dev areas: Mobile apps
```
Comment (justinegeffen) [inline site/content/contribute/getting-started/core-committers.md:29]:
```suggestion
    - Dev areas: Mobile app
```

File: site/content/contribute/getting-started/core-committers.md [hunk 2]
```diff
@@ -67,7 +67,7 @@ Below is the list of core committers working on Mattermost:
     - Dev areas: Desktop / Web and Mobile apps
 - **<a name="miguel.alatzar">Miguel Alatzar</a>**
     - @miguel.alatzar on [community.mattermost.com](https://community.mattermost.com/core/messages/@miguel.alatzar) and [@migbot](https://github.com/migbot) on GitHub
-    - Dev areas: Mobile / Desktop and Web apps
+    - Dev areas: Mobile apps
```
Comment (justinegeffen) [inline site/content/contribute/getting-started/core-committers.md:70]:
```suggestion
    - Dev areas: Mobile app
```

File: site/content/contribute/getting-started/core-committers.md [hunk 3]
```diff
@@ -164,6 +164,12 @@ Below is the list of core committers working on Mattermost:
 - **<a name="rvillablanca">Rodrigo Villablanca</a>**
     - @rvillablanca on [community.mattermost.com](https://community.mattermost.com/core/messages/@rvillablanca) and [@rvillablanca](https://github.com/rvillablanca) on GitHub
     - Dev areas: Server, Store Layer
+- **<a name="shaz.amjad">Shaz Amjad</a>**
+    - @shaz.amjad on [community.mattermost.com](https://community.mattermost.com/core/messages/@shaz.amjad) and [@shazm](https://github.com/shazm) on GitHub
+    - Dev areas: Mobile apps
```
Comment (justinegeffen) [inline site/content/contribute/getting-started/core-committers.md:169]:
```suggestion
    - Dev areas: Mobile app
```

File: site/content/contribute/getting-started/core-committers.md [hunk 4]
```diff
@@ -164,6 +164,12 @@ Below is the list of core committers working on Mattermost:
...
+- **<a name="avinash.lingaloo">Avinash Lingaloo</a>**
+    - @avinash.lingaloo on [community.mattermost.com](https://community.mattermost.com/core/messages/@avinash.lingaloo) and [@avinashlng1080](https://github.com/avinashlng1080) on GitHub
+    - Dev areas: Mobile apps
```
Comment (justinegeffen) [inline site/content/contribute/getting-started/core-committers.md:172]:
```suggestion
    - Dev areas: Mobile app
```

**Review by justinegeffen — APPROVED** [review]

**Comment by justinegeffen** [comment]
Thanks, @enahum. I've made one suggestion but otherwise LGTM. :)

File: site/content/contribute/getting-started/core-committers.md (hunk 1 above)
Comment (enahum) [reply site/content/contribute/getting-started/core-committers.md:29]: Well there is more than one, I'll change it to just mobile

**Pull request merged**